lib/gosu.rb

```diff
@@ -245,4 +259,28 @@ def self.blend_mode(mode)
       raise ArgumentError, "No such blend mode: #{mode}"
     end
   end
+
+  def self.check_last_error
+    if (err = Gosu_last_error())
+      raise err
+    end
+  end
 end
+
+# Individual classes need to be loaded after defining Gosu.check_last_error.
```

Comment:
:/

Gosu::Color::RED calls Gosu::Color#initialize, which calls Gosu.check_last_error. Not sure if there is a better way around this.